aws/provision.sh

```diff
@@ -14,6 +14,35 @@ chmod 400 ~/.ssh/id_rsa.pub
 eval `ssh-agent`
 ssh-add ~/.ssh/id_rsa
 
+# TODO: download bosh-init from our own bucket
```

Comment:
Slow & they might remove it. It's better to manage this yourself where you have full control.